modyn/config/schema/pipeline.py

```diff
@@ -364,17 +362,16 @@ def validate_gpus(cls, value: int) -> int:
         return value
 
     @model_validator(mode="after")
-    @classmethod
-    def validate_num_samples_to_pass(cls, data: "TrainingConfig") -> "TrainingConfig":
-        if data.initial_model == "pretrained":
-            if not data.use_previous_model:
-                raise ValidationError(
+    def validate_pretrained(self) -> Self:
+        if self.initial_model == "pretrained":
```

Comment:
the previous name is confusing as this method is not acted on the field `num_samples_to_pass`

Reply:
Good catch! :) That might have been a copy & paste issue